src/nouveau/src/nouveau_fabric_search.erl

```diff
@@ -145,6 +145,9 @@ merge_search_results(A, B, #state{} = State) ->
         ),
         <<"ranges">> => merge_facets(
             maps:get(<<"ranges">>, A, null), maps:get(<<"ranges">>, B, null)
+        ),
+        update_latency => max(
```

Comment:
It looks a bit odd that this field is an atom while all the others are binaries. Should we make them all the same (either all atoms or all binaries).

Reply:
agreed it looks odd, but my reasoning is this: the binaries came from the jvm side, as a consequence of being marshalled to/from json. the atoms are purely internal (update_latency is calculated on the erlang side).

Reply:
that's fair, it's good reason to keep it as is then